README.md

```diff
@@ -93,6 +93,41 @@ For Linux:
 You can also choose between different executable ruby versions by `rvm use <version>` .
 The current default is 2.5.
 
+
+### As a CI System Container
+
+The OCI images provided by rake-compiler-dock can be used without the `rake-compiler-dock` gem or wrapper. This may be useful if your CI pipeline is building native gems.
+
+For example, a Github Actions job might look like this:
+
+``` yaml
+jobs:
+  native-gem:
+    name: "native-gem"
+    runs-on: ubuntu-latest
+    container:
+      image: "larskanis/rake-compiler-dock-mri-x86_64-linux:1.1.0"
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./scripts/build-native-gem
+      - uses: actions/upload-artifact@v2
+        with:
+          name: native-gem
+          path: gems
+          retention-days: 1
+```
+
+And the script being run in the container already has a Ruby build environment set up, and could be as simple as:
+
+``` sh
+#! /usr/bin/env bash
+bundle install
+bundle exec rake gem:x86_64-linux:buildit
+```
+
+For an example of rake tasks that support this style of invocation, visit https://github.com/flavorjones/ruby-c-extensions-explained/tree/main/precompiled
+
```

Comment:
Can't we make the description self-contained? The rake task `gem:x86_64-linux:buildit` isn't defined here and relying on a different project is precarious. One shouldn't need to study a second project in this case.

Reply:
On the other hand a link to a second project as an example and for deeper understanding is OK.

Reply:
Done! Force-pushed documentation that I hope is clearer and more self-contained. Please let me know what you think.